transcom/mymove
# B 20211 int remove coast guard ff

## Agility ticket 
https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A964023

## Original story that put FF in
https://github.com/transcom/mymove/pull/12772

## Summary
It was discovered that we no longer need a FF to wrap the EMPLID field for the coast guard customer.  The field is ready to display and use.  We just need a way to make the DB field non-unique if on loadtest or staging, for local testing purposes. 

### How to test

1. Access the milmove app as a customer, create a new customer.
2. Choose Coast Guard as the branch of service.
3. Verify that you can type in an existing emplid that already exists for another milmove user and save.
4. Verify that you can also go under Edit Profile, Edit Service Info section and update the EMPLID field to be something that already exists in the DB by another user.


You can verify that the migration file (20240611185048_drop_unique_constraint_emplid.up.sql) to drop the constraint is loaded in the S3 bucket on loadtest and staging by running this command and see the file listed.
DISABLE_AWS_VAULT_WRAPPER=1 AWS_PROFILE=transcom-gov-id AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-milmove-stg -- aws s3 ls s3://transcom-gov-milmove-stg-app-us-gov-west-1/secure-migrations/

You can verify that a blank migration file is loaded to S3 for prod but running this command and see the 0 file size listed.
DISABLE_AWS_VAULT_WRAPPER=1 AWS_PROFILE=transcom-gov-id AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-milmove-prd -- aws s3 ls s3://transcom-gov-milmove-prd-app-us-gov-west-1/secure-migrations/
